all-contacts/1-pdb-generation/fah-pdbs-make.pl

```diff
@@ -76,14 +84,14 @@ sub generate_pdbs_from_logfile {
 
         #TODO: Comment on what `echo 1 1` is for
```

Comment:
Artifact?

Reply:
I need to verify that selecting 1 then another 1 is the correct option for `trjconv` from GROMACS 3.3.

Reply:
I will update this comment.